explorer/frontend/api/transactions.js

```diff
@@ -336,10 +336,10 @@ const formatMultisigTransaction = (data, filter) => {
 	const formattedEmbeddedTransaction = transactionFromDTO(rawEmbeddedTransaction, filter.address);
 
 	// Fees breakdown
-	const multisigFee = truncateDecimals(formattedTransaction.fee, config.NATIVE_MOSAIC_DIVISIBILITY);
-	const embeddedTransactionsFee = truncateDecimals(formattedEmbeddedTransaction.fee, config.NATIVE_MOSAIC_DIVISIBILITY);
+	const multisigFee = truncateDecimals(formattedTransaction.fee || 0, config.NATIVE_MOSAIC_DIVISIBILITY);
+	const embeddedTransactionsFee = truncateDecimals(formattedEmbeddedTransaction.fee || 0, config.NATIVE_MOSAIC_DIVISIBILITY);
 	const signaturesFee = truncateDecimals(
-		data.embeddedTransactions[0].signatures.reduce((total, signature) => total + signature.fee, 0),
+		data.embeddedTransactions[0].signatures.reduce((total, signature) => total + (signature.fee || 0), 0),
```

Comment:
Do you know the reason why the fee is missing?

Reply:
Seems like unconfirmed transactions have missing fee field. cc: @AnthonyLaw

Reply:
I think this change should be on the backend for the fix.

@AnthonyLaw, is this related to WebSockets?

Reply:
@Wayonb I removed the fallback.